webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Overview/CardsRegistry.js

```diff
@@ -12,6 +12,6 @@ const cards = [
 
 export const registerCoreCards = () => {
   cards.forEach(({ key, Component, weight }) => {
-    addGlobalFill('details-cards', key, <Component key={key} />, weight);
+    addGlobalFill('host-overview-cards', key, <Component key={key} />, weight);
```

Comment:
Since you filed PRs changing plugins, is this a breaking change? That would imply plugins can't release for multiple Foreman versions. Is that intended?